Sample/SampleForageSDK/ViewController.swift

```diff
@@ -30,11 +30,15 @@ class ViewController: UIViewController {
 
         ClientSharedData.shared.merchantID = merchantID
         ClientSharedData.shared.sessionToken = sessionToken
-
+        
+        /// CHANGE WITH CAUTION
+        /// we intentionall call ``ForageSDK.setup`` with invalid values first to ensure that we can update
+        /// the config values later on in the ``CardNumberViewController.viewDidAppear`` method
         ForageSDK.setup(
             ForageSDK.Config(
-                merchantID: merchantID,
-                sessionToken: sessionToken
+                merchantID: "invalid mid/...",
+                // intentionally omitted environment prefix from this sessionToken
+                sessionToken: "invalid session Token with no environment prefix"
             )
         )
```

Comment:
We can also add a test case in the Sample app that simply goes back and re-submits new session token + merchant ID, but that will take some time to introduce

So in the meantime, let's keep this code - unless @dleis612 objects